docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -48,14 +48,31 @@ You need to add the following configuration items to the `core-site.xml` configu
   <description>The Alluxio AbstractFileSystem (Hadoop 2.x)</description>
 </property>
 ```
-If your Alluxio is HA, another configuration need to added:
+
+To use fault tolerant mode, set the Alluxio cluster properties appropriately in an
+`alluxio-site.properties` file which is on the classpath.
+
+```properties
+alluxio.zookeeper.enabled=true
+alluxio.zookeeper.address=[zookeeper_hostname]:2181
+```
+
+Alternatively you can add the properties to the Hadoop `core-site.xml` configuration which is then
```

Comment:
The previous configuration values were also set in `core-site` and we copy over Hadoop configuration to Alluxio configuration for the Hadoop API